### DISCUSSION TITLE
Copilot not working

### DISCUSSION THREAD

**Discussion by battlelinegames** [community/community#45274] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I've signed up for copilot and installed the VS Code plugin, but nothing seems to happen.  The tutorials I've seen fill in code as you type, but I just don't see anything, and I don't know why.  I've tried messing with the settings, disabling and re-enabling the plugin, and restarting VS CODE.  I received an email from GitHub confirming that I've signed up for it.  I don't see any message indicating why it would not be working.  I'm trying to use it in a new .js file and nothing is happening as far as I can tell.

**Comment by battlelinegames** [comment]
For the first time, I had an error appear in my output:

GitHub Copilot could not connect to server. Extension activation failed: "<semantic> TypeScript Server Error (5.0.0-dev.20230123)

Does anyone know what that is about?

**Reply by dbeberman** [reply]
I have a similar problem.  I'm on AWS Workspace -- VS Code is version: 1.74.3.
I signed up for a copilot license.
Then tried to install from the VS Code marketplace as directed.  It does a popup to login to github.com, even though I was already logged in.  Brings up the browser (Firefox), I do the login.  Then VS Code simply core dumps.
I'm tried the uninstall/reinstall, even reinstalled VS Code.  
I have just simply restarted VS Code.  In that case, the Copilot icon, bottom right, is a busy waiting circle, and there are no language completions.
Would like to try this out.  I'm currently programming in Go and Python.
If anyone has an answer: email is dbeberman@gmail.com

**Comment by SakibSumon** [comment]
did you figure out your problem? copilot sometimes takes longer than usual to initiate for the first time or in a new code editor.

**Reply by dbeberman** [reply]
i gave up, cancelled with OpenAI and uninstalled the Copilot extension in VS code for now.  Don't have time to deal with it at the moment.

**Comment by battlelinegames** [comment]
No.  I did start getting error messages in vs code.  Will send the. When
I'm back at my desk

**Comment by battlelinegames** [comment]
Right now, in VS Code plugin Runtime Status tab it says "Not yet activated"

I activated it in my github account so I'm not sure why it doesn't
recognize that.

Thanks

**Comment by SakibSumon** [comment]
you should contact their [support](https://support.github.com/request/actions?tags=hubberfy_actions) if you haven't already. they're very responsive. you can explain your issue to them and hopefully, get your solution.
